src/main/java/net/malisis/core/client/gui/GuiRenderer.java

```diff
@@ -334,9 +336,38 @@ public String clipString(String text, int width)
 		int lineWidth = 0;
 		int wordWidth = 0;
 		int index = 0;
+
+		StringBuilder currentColorCode = new StringBuilder();
+		boolean skipLine = false;
+		boolean prefixColorCode = false;
+
 		while (index < text.length())
 		{
 			char c = text.charAt(index);
+
+			if (line.length() == 0) {
+				if (prefixColorCode) {
+					if (!skipLine) {
+						line.insert(0, currentColorCode.toString(), 0, currentColorCode.length());
+					} else {
+						skipLine = false;
+					}
+				}
+			}
+
+			//Figure out if a valid character code is being presented
+			if (c == CODE_COLOR_BEGIN) {
```

Comment:
@Ordinastie 

This bit of code works fine unless you want to combine two color codes for a line (Such as RED + UNDERLINE). We can work on a solution unless you have something better in mind.